.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+### INCLUDED CHANGES:
+-
+
+### TODO:
+- [ ] CHANGELOG(s) updated with appropriate info
```

Comment:
maybe

```suggestion
- [ ] crate(s) CHANGELOG(s) updated with appropriate info
```

just to highlight that the top-level CHANGELOG isn't to be udpated manually? i'm also fine merging as is

Reply:
I think I'm going to merge as-is so the wording is less confusing... but if people are having trouble with this concept, we could add an auto-generated notification to the top level changelog to discourage direct edits.